client/my-sites/plans-grid/components/header-price.tsx

```diff
@@ -42,10 +46,13 @@ const Badge = styled.div< { isForIntroOffer?: boolean; isHidden?: boolean } >`
 	visibility: ${ ( { isHidden } ) => ( isHidden ? 'hidden' : 'visible' ) };
 `;
 
-const HeaderPriceContainer = styled.div`
+const HeaderPriceContainer = styled.div< { discounted?: boolean } >`
 	padding: 0 20px;
 	margin: 0 0 4px 0;
-
+	min-height: ${ ( { discounted } ) => ( discounted ? '96.5px' : 'auto' ) };
```

Comment:
Dang 😆

Reply:
Side note:

One thing I noticed with styled components while working on the plan type selector is that the inline styling circumvents the linter guards that we have in place in our scss files ( Ex. using rem instead of px for font-sizes, etc. )

Is there an easy way to double-check this before committing code @jdc91?

Reply:
Whether or not the inline styling we're introducing with styled components would violate any styling conventions we've established with Stylelint

Here's an earlier example

https://github.com/Automattic/wp-calypso/blob/9e09ad1b28282f53112e9167e98b03f04b26bfad/packages/plans-grid-next/src/components/plan-type-selector/components/interval-type-dropdown.tsx#L15

If used in an scss file, it triggers the `Unexpected unit "px" for property "font-size" (declaration-property-unit-allowed-list)` warning, but doesn't when used within a styled component